packages/dx-scheduler-core/src/utils.js

```diff
@@ -0,0 +1,50 @@
+import moment from 'moment';
+
+const createExcludedInterval = (day, start) => {
+  const leftBound = moment(start.day(day));
+  return [
+    leftBound,
+    moment(leftBound).hour(start.hour()).endOf('day'),
+  ];
+};
+
+const excludedIntervals = (excludedDays, start) => excludedDays
+  .map(day => (day === 0 ? 7 : day))
+  .sort()
```

Comment:
Please note that `Array.prototype.sort()` default sort order is based on string Unicode codes, so it won't work for numbers correctly. See more details [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort).